core/templates/components/forms/custom-forms-directives/audio-file-uploader.component.ts

```diff
@@ -24,19 +24,21 @@ import { downgradeComponent } from '@angular/upgrade/static';
   templateUrl: './audio-file-uploader.component.html'
 })
 export class AudioFileUploaderComponent {
-  @Input() droppedFile: FileList;
+  // This property is initialized using angular lifecycle hooks, see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1.
```

Comment:
Use the same format as in the other PRs.

Reply:
done